### DISCUSSION TITLE
\n feature and Quotes for Python and C++

### DISCUSSION THREAD

**Discussion by glennedstrom** [monkeytypegame/monkeytype#827] [Ideas] [answered]
I would like to be able to at least be able to have \n work whenever you press the enter keys (and possibly \t when you press tab even though you have it mapped for something else and you can have a setting to toggle the tab key) because then I could put algorithms in to practice my programming speed, without all of my code being a one-liner wrapped. If you get that working then add some c++ quotes or sections of code that do things like calculate the sum of an array, or define a binary tree node, or other related basic things that you can find on most coding websites. I could do the quotes part for sure but I'm not sure where to look for the \n yet and also maybe \t

**Comment by Miodec** [comment, marked as answer]
\n support should be live now
